brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py

```diff
@@ -27,7 +27,7 @@ def create_atlas(working_dir, resolution):
         "Lazcano, I. et al. 2021, https://doi.org/10.1038/s41598-021-89357-3"
     )
     ATLAS_FILE_URL = "https://zenodo.org/records/4595016"
-    ORIENTATION = "ipl"
+    ORIENTATION = "lpi"
```

Comment:
```suggestion
    ORIENTATION = "ipl"
```
This will be fixed in a separate issue #339